### ISSUE TITLE
all master* in `.fits`

### ISSUE THREAD

**Issue by londumas** [igmhub/SaclayMocks#13]
This is not that important but would simplify my life a lot.
Since master files produced by LyaCoLoRe have to be in `.fits` because of a known bug in fitsio, could you also have all master files in `.fits`?
Simply compare:
```
ls -ltr /global/projecta/projectdirs/desi/mocks/lya_forest/london/v6.0/v6.0.0/master*
```
to
```
ls -ltr /global/projecta/projectdirs/desi/mocks/lya_forest/saclay/v4.1/master/*
```

**Comment by jmarclegoff** [comment]
@londumas You just mean that we should not use .fits.gz that fitsio is supposed to read but there is a bug ?

**Comment by londumas** [comment]
@jmarclegoff, I am simply talking about the master files, not the transmission files. And indeed there is a bug in fitsio when the size of the file is too big, which happens for the random catalog of quasars in LyaCoLoRe: https://github.com/esheldon/fitsio/issues/199

**Comment by jmarclegoff** [comment]
I have unzip the master files in v4.1. 
@TEtourneau you should change your scripts so that further master files will not be gzip.
@londumas so we should not gzip the master files, which can be 2.5GB big, but we can gzip transmission files, which are less than 100MB ? This is good since 
- there are hundreds of them and at the end they represent much more than the master files
- gzip is very efficient on the transmission file, gain by a factor close to 7. Although the gain will be smaller once we set z_QSO_min = 1.8 instead of 1.3.

Do you know the maximum size for which we can use gzip ?
According to Erin Sheldon the problem is not in his interface but in cfitsio itself. I have used cfitsio in C in the past, for the BOSS mocks. May be I should put on my todo list writing a C code that fails in reading a large fits.gzip file to ask cfitsio maintainers to solve the issue.

**Comment by TEtourneau** [comment]
@londumas Sure. I made the changed for the master files: they are all in .fits
I also changed the transmission files to be in .fits.gz

**Comment by londumas** [comment]
fixed in v4.2.

```
-rw-rw----   1 tetourne desi  179248320 Mar 19 04:35 master.fits
-rw-rw----   1 tetourne desi 1716782400 Mar 19 04:37 master_randoms.fits
-rw-rw----   1 tetourne desi   16727040 Mar 19 05:11 master_DLA.fits
-rw-rw----   1 tetourne desi   26758080 Mar 19 05:11 master_DLA_randoms.fits
```
